docs/en-US/component/descriptions.md

```diff
@@ -1,201 +1,56 @@
-## Descriptions
+# Descriptions
 
 Display multiple fields in list form.
 
-### Basic usage
+<style lang="scss">
```

Comment:
Should we add `scoped` here? Since if we visit this page then visit other page we might pollute other page's style